data/moves.js

```diff
@@ -13796,7 +13796,13 @@ let BattleMovedex = {
 		effect: {
 			duration: 1,
 			onResidualOrder: 20,
-			// implemented in BattlePokemon#getTypes
```

Comment:
My best guess is that this line shouldn't have been commented out, because a lot of the implementation is still in `getTypes`.